manager/src/main/java/org/apache/hertzbeat/manager/controller/CollectorController.java

```diff
@@ -112,23 +85,25 @@ public ResponseEntity<Message<Void>> offlineCollector(
     public ResponseEntity<Message<Void>> deleteCollector(
             @Parameter(description = "collector name", example = "demo-collector")
             @RequestParam(required = false) List<String> collectors) {
-        this.collectorService.deleteRegisteredCollector(collectors);
-        return ResponseEntity.ok(Message.success("Delete success"));
+        try {
+            this.collectorService.deleteRegisteredCollector(collectors);
+            return ResponseEntity.ok(Message.success("Delete success"));
+        } catch (CommonException ex) {
```

Comment:
This can be removed because `CommonException` is handled by `GlobalExceptionHandler`.

Reply:
Thanks, fixed.